test/unit/extract/phmsagas_test.py

```diff
@@ -8,20 +6,20 @@
 
 
 class FakeExtractor(Extractor):
-    def __init__(self):
+    def __init__(self, mocker):
```

Comment:
See above non-blocking comment about passing in the actual dependencies vs. passing in `mocker`.